Sources/KlaviyoSwift/KlaviyoEnvironment.swift

```diff
@@ -12,7 +12,7 @@ import UIKit
 
 var environment = KlaviyoEnvironment.production
 
-let PRODUCTION_HOST = Klaviyo.sharedInstance.KlaviyoServerURLString
+let PRODUCTION_HOST = "https://a.klaviyo.com"
```

Comment:
Wait why is this changing? You didnt change that constant

Reply:
This will eventually go away and it's not needed anymore. Changing host is now accomplished through a different mechanism.